packages/sdk/.npmignore

```diff
@@ -0,0 +1,9 @@
+# Exclude source maps
+*.map
+**/*.map
+
+# Exclude compressed files
```

Comment:
Didn't realize we'd be generating/publishing these, does seem odd to be including them 🤔

Reply:
yeahhh 😅